src/components/Radiobutton.js

```diff
@@ -13,6 +13,11 @@ export const Radiobutton = ({ title, disabled, selected, onPress, containerStyle
     onPress={onPress}
     size={normalize(24)}
     containerStyle={[styles.containerStyle, containerStyle]}
+    accessibilityLabel={`${
+      selected
+        ? texts.accessibilityLabels.checkbox.active
+        : texts.accessibilityLabels.checkbox.inactive
+    } ${title}`}
```

Comment:
isn't one of the words missing brackets, so voice over is reading it too fast without a gap? i think adding them like `(${title})` will force it.

Reply:
Yes, it makes sense. I fixed it.

https://github.com/smart-village-solutions/smart-village-app-app/commit/f425b1ee2acfc59d90979d9e08ec3fa64b692c06